shared/cfg/machines.cfg

```diff
@@ -16,6 +16,14 @@ variants:
             cd_format = ahci
         enable_pvpanic = yes
         ioport_pvpanic = 0x505
+        pci_controllers = root_port pcie_switch dmi2pci_bridge pci_bridge
+        pci_bus = pci.0
+        type_root_port = ioh3420
+        type_pcie_switch = x3130
+        pci_bus_pcie_switch = root_port
+        type_dmi2pci_bridge = i82801b11-bridge
+        type_pci_bridge = pci-bridge
+        pci_bus_pci_bridge = dmi2pci_bridge
```

Comment:
This comment is outdated, we decide somewhere else that adjusting q35 is the way to go as it's the recommended solution for q35 machines. (just repeating it here so people are not confused)